tests/resources/Nimbus-Util.robot

```diff
@@ -571,3 +572,17 @@ Get Name of First Local Storage For Host
     ${datastores}=  Run  govc host.info -host ${host} -json | jq -r '.HostSystems[].Config.FileSystemVolume.MountInfo[].Volume | select (.Type\=\="VMFS") | select (.Local\=\=true) | .Name'
     @{datastores}=  Split To Lines  ${datastores}
     [Return]  @{datastores}[0]
+
+# Simple wrapper to Wait Until Keyword Succeeds that allows callers to:
+# * use default retry count and delay
+# * specify specific retry counts and delays
+# * robot framework executor to override both the above via the following environment variables:
```

Comment:
nit: This documentation reads a bit awkwardly. ("Simple wrapper to Wait Until Keyword Succeeds that allows callers to ... robot framework executor to override ...")